### PR TITLE
Support csv output

### PR DESCRIPTION
**Pull request by gky360**
Support output as csv format.

### CODE REVIEW

**Review by makky3939 — APPROVED** [review]
👍

**Review by niboshi — APPROVED** [review]
Thank you!

File: bin/licensetool [hunk 1]
```diff
@@ -3,23 +3,35 @@
...
+const { stringify } = require('csv-stringify/sync');
 const { strict: assert } = require('node:assert');
 
 const licenseChecker = require('license-checker-rseidelsohn');
 const spdxExpressionParse = require('spdx-expression-parse');
 const spdxLicenseList = require('spdx-license-list/full');
 
 function parseArgs() {
-  // TODO: Use node:util.parseArgs
   // Currently only a limited syntax is accepted: "format --config <path>"
```
Comment (niboshi) [inline bin/licensetool]: I think the second line of this comment is a little bit outdated, too.
You could just remove it as well.

**Review by niboshi — APPROVED** [review]

File: bin/licensetool (hunk 1 above)
Comment (gky360) [reply bin/licensetool]: Thank you. I removed the comment.

**Pull request merged**